### PR TITLE
🌐 Add Korean translation for `ko/docs/advanced/response-headers.md`

### PR DESCRIPTION
**Pull request by kwang1215**
Add Korean translation for `ko/docs/advanced/response-headers.md`

### CODE REVIEW

File: docs/ko/docs/advanced/response-headers.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**FastAPI**는 해당 *임시* 응답에서 헤더(쿠키와 상태 코드도 포함)를 추출하여, 여러분이 반환한 값을 포함하는 최종 응답에 `response_model`로 필터링된 값을 넣습니다.
+
+또한, 종속성에서 `Response` 매개변수를 선언하고 그 안에서 헤더(및 쿠키)를 설정할 수 있습니다.
+
+## `Response` 직접 반환하기
+
+`Response`를 직접 반환할 때에도 헤더를 추가할 수 있습니다.
+
+[Return a Response Directly](response-directly.md){.internal-link target=_blank}에서 설명한 대로 응답을 생성하고, 헤더를 추가 매개변수로 전달하세요.
```
Comment (Limsunoh) [inline docs/ko/docs/advanced/response-headers.md:25]:
```suggestion
[응답을 직접 반환하기](response-directly.md){.internal-link target=_blank}에서 설명한 대로 응답을 생성하고, 헤더를 추가 매개변수로 전달하세요.
```
'Return a Response Directly' If we change this part to '응답을 직접 반환하기' the translation might look clearer

Comment (kwang1215) [reply docs/ko/docs/advanced/response-headers.md]: Thank you for your review!

**Comment by Limsunoh** [comment]
LGTM

**Review by Limsunoh — APPROVED** [review]
LGTM

**Review by 9zimin9 — APPROVED** [review]
LGTM !

**Comment by alejsdev** [comment]
Thanks for your contribution! @kwang1215 :nerd_face: 
And thanks to everyone for the help and reviews :rocket:

**Pull request merged**